{
	"schema_version": "1.6.0",
	"id": "RLSA-2021:4149",
	"modified": "2023-02-02T14:11:33.714070Z",
	"published": "2021-11-09T08:24:34Z",
	"related": [
		"CVE-2020-35653",
		"CVE-2020-35655",
		"CVE-2021-25287",
		"CVE-2021-25288",
		"CVE-2021-25290",
		"CVE-2021-25292",
		"CVE-2021-25293",
		"CVE-2021-27921",
		"CVE-2021-27922",
		"CVE-2021-27923",
		"CVE-2021-28675",
		"CVE-2021-28676",
		"CVE-2021-28677",
		"CVE-2021-28678",
		"CVE-2021-34552"
	],
	"summary": "Moderate: python-pillow security update",
	"details": "The python-pillow packages contain a Python image processing library that provides extensive file format support, an efficient internal representation, and powerful image-processing capabilities.\n\nSecurity Fix(es):\n\n* python-pillow: Out-of-bounds read in J2K image reader (CVE-2021-25287)\n\n* python-pillow: Out-of-bounds read in J2K image reader (CVE-2021-25288)\n\n* python-pillow: Negative-offset memcpy in TIFF image reader (CVE-2021-25290)\n\n* python-pillow: Regular expression DoS in PDF format parser (CVE-2021-25292)\n\n* python-pillow: Out-of-bounds read in SGI RLE image reader (CVE-2021-25293)\n\n* python-pillow: Excessive memory allocation in BLP image reader (CVE-2021-27921)\n\n* python-pillow: Excessive memory allocation in ICNS image reader (CVE-2021-27922)\n\n* python-pillow: Excessive memory allocation in ICO image reader (CVE-2021-27923)\n\n* python-pillow: Excessive memory allocation in PSD image reader (CVE-2021-28675)\n\n* python-pillow: Infinite loop in FLI image reader (CVE-2021-28676)\n\n* python-pillow: Excessive CPU use in EPS image reader (CVE-2021-28677)\n\n* python-pillow: Excessive looping in BLP image reader (CVE-2021-28678)\n\n* python-pillow: Buffer overflow in image convert function (CVE-2021-34552)\n\n* python-pillow: Buffer over-read in PCX image reader (CVE-2020-35653)\n\n* python-pillow: Buffer over-read in SGI RLE image reader (CVE-2020-35655)\n\nFor more details about the security issue(s), including the impact, a CVSS score, acknowledgments, and other related information, refer to the CVE page(s) listed in the References section.\n\nAdditional Changes:\n\nFor detailed information on changes in this release, see the Rocky Linux 8.5 Release Notes linked from the References section.",
	"severity": [
		{
			"type": "CVSS_V3",
			"score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H"
		}
	],
	"affected": [
		{
			"package": {
				"ecosystem": "Rocky Linux:8",
				"name": "python-pillow",
				"purl": "pkg:rpm/rocky-linux/python-pillow?distro=rocky-linux-8-5-legacy&epoch=0"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "0:5.1.1-16.el8"
						}
					]
				}
			],
			"database_specific": {
				"source": "https://storage.googleapis.com/resf-osv-data/RLSA-2021:4149.json"
			}
		}
	],
	"references": [
		{
			"type": "ADVISORY",
			"url": "https://errata.rockylinux.org/RLSA-2021:4149"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1915420"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1915432"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1934685"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1934699"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1934705"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1935384"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1935396"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1935401"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958226"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958231"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958240"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958252"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958257"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1958263"
		},
		{
			"type": "REPORT",
			"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1982378"
		}
	],
	"credits": [
		{
			"name": "Rocky Enterprise Software Foundation"
		},
		{
			"name": "Red Hat"
		}
	]
}
